packages/react-scripts/template/src/index.js

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import * as serviceWorkerRegistration from './service-worker-registration';
```

Comment:
I think you are importing both `register` and `unregister` methods so they can easily unregister the SW if its in production and has already been registered by users. I think this is an unlikely situation, considering this is the beginning of a react app.
IMHO it would be better to `export default register` and then just do `import registerSW from './service-worker-registration'` and `registerSW();`